frontend/src/pages/projects/update/[user]/[projectName].js

```diff
@@ -7,7 +7,7 @@ import FilesPreview from '@components/FilesPreview'
 import useForm from '@hooks/useForm'
 import { ProjectUpdateForm } from '@models/ProjectUpdateForm'
 import { UploadContext } from '@contexts/UploadContext'
-import { getRepo, updateRepo } from '@utils/giteaApi'
+import { getRepo as getProject, updateRepo } from '@utils/giteaApi'
```

Comment:
![image](https://www.reactiongifs.com/r/but-why.gif)

Reply:
:rofl: :rofl: :rofl:
I think it makes more sense; in the pages, the repo is always referenced as a project, though In the API client it's explicit what endpoint we are hitting

Reply:
I think it's and extra level of renaming that makes it less clear. Use the API you are writing (the Gitea client). If you don't like it, write a better API.